Firebase/Auth/Source/FIRAuth.m

```diff
@@ -1196,6 +1194,48 @@ - (void)phoneNumberSignInWithRequest:(FIRVerifyPhoneNumberRequest *)request
 }
 #endif
 
+/** @fn internalSignInAndRetrieveDataWithCustomToken:completion:
+    @brief Signs in a Firebase user given a custom token.
+    @param completion A block which is invoked when the custom token sign in request completes.
+ */
+- (void)internalSignInAndRetrieveDataWithCustomToken:(NSString *)token
+                           completion:(nullable FIRAuthDataResultCallback)completion {
+  FIRVerifyCustomTokenRequest *request =
+    [[FIRVerifyCustomTokenRequest alloc] initWithToken:token
+                                  requestConfiguration:_requestConfiguration];
+  [FIRAuthBackend verifyCustomToken:request
+                           callback:^(FIRVerifyCustomTokenResponse *_Nullable response,
+                                      NSError *_Nullable error) {
+    if (error) {
+      completion(nil, error);
```

Comment:
completion block is nullable, please verify it's not null

Reply:
done.